src/plugins/mer/merbuildengineoptionspage.cpp

```diff
@@ -41,7 +43,7 @@ MerBuildEngineOptionsPage::MerBuildEngineOptionsPage(QObject *parent)
     : IOptionsPage(parent)
 {
     setCategory(Core::Id(Constants::MER_OPTIONS_CATEGORY));
-    setDisplayCategory(QCoreApplication::translate("Mer", Constants::MER_OPTIONS_CATEGORY_TR));
+    setDisplayCategory(Sfdk::Sdk::osVariant());
```

Comment:
Please always prefer `using namespace` at file scope. Many occurrences elsewhere.

Reply:
Fix for all, but why? Use for one use is it good?

Reply:
https://doc-snapshots.qt.io/qtcreator-extending/coding-style.html#namespaces "In other cases, you are encouraged to use using-directives, as they help you avoid cluttering the code."

The less exceptions exist to a rule the easier it is to maintain compliance with the rule. It often starts with just one occurrence :)